.github/ISSUE_TEMPLATE/enhancement-tracking.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Enhancement Tracking Issue
+about: Provide supporting details for a feature in development
+labels: feature
+title: '[Enhancement]: <title>'
+
+---
+<!-- Feature requests are unlikely to make progress as an issue.
+
+A proposal that works through the design along with the implications of the change should be submitted as a design document in https://github.com/redhat-et/microshift/tree/main/doc/design/
+
+-->
+#### Design Document Link
```

Comment:
We'll definitely want a design doc whenever an enhancement impacts MicroShift scope, UX (incl. CLI options and externally visible behaviour), or architecture.

If this is not the case, do we want to allow bypassing the design doc in favour of discussion and agreement in the issue? If so, I propose moving this to the bottom and asking whether one of the above impacts is true and if so, then ask for link.

Reply:
> ... do we want to allow bypassing the design doc in favour of discussion and agreement in the issue? If so, I propose moving this to the bottom and asking whether one of the above impacts is true and if so, then ask for link.

I think 99% of enhancements are going to affect these critical criteria just by definition of being an enhancement.  So having an Enhancement issue, with the design doc link and comment as-is,  will help drive them towards that without us having to repeatedly ask contributors to start a doc.  It also gives us an escape hatch in (probably rare) cases where we decide ad hoc that the enhancement _doesn't_ warrant a full blown doc.

Reply:
Ok, that makes sense to me. We can still modify later if needed.